docs/UserGuide.md

```diff
@@ -270,18 +298,32 @@ To find the supplier whose name contains "link" and company contains "NU":
 
 ### Sort suppliers: `sort -s`
 
-The `sort -s` command is used to sort suppliers in VendorVault.
-This helps you to view the suppliers in a different order (ascending or descending), based on the supplier name.
+The `sort -s` command is used to sort suppliers according to the supplier name in Vendor Vault.
+This helps you to view the suppliers in a different order (ascending or descending).
 
 Format: `sort -s so/SORT_ORDER sb/SORT_BY`
-- `SORT_ORDER`: Must be either 'a' for ascending or 'd' for descending.
+- `SORT_ORDER`: Must be either 'a' for ascending or 'd' for descending. 
 - `SORT_BY`: Must be 'n' for name. (Current version of VendorVault only supports sorting by name)
 
+<box type="warning" seamless>
+
 **Warnings**:
-- A spacing between `add` and `-s` is compulsory
-- All prefixes and parameters must be given
-- No duplicate prefix can be used
-- Parameters used are **case-sensitive**
+- A spacing between `add` and `-s` is compulsory.
+- A spacing between `-s` and `so` is compulsory.
+- A spacing between `SORT_ORDER` and `sb` is compulsory.
+- All prefixes and parameters must be given.
+- No duplicate prefix can be used.
+- Parameters used are **case-sensitive**.
+</box>
+
+<box type="tip" seamless>
+
+**Tip:**
+- The sort command will only sort all suppliers in VendorVault.
+- eg: 
+    - If you have searched for a supplier using the `find` command, 
+  the sort command will not sort the results of the `find` command.
```

Comment:
Perhaps changing the results to 'previous results' would make it clearer to the users